### ISSUE TITLE
[BUG][opensearch] unused readinessProbe in values.yaml

### ISSUE THREAD

**Issue by rursprung** [opensearch-project/helm-charts#194]
**Describe the bug**
the following definition from the `values.yaml` (same also in the three CI-related value yaml files) is unused:
https://github.com/opensearch-project/helm-charts/blob/63847ee78834063f5f79d2f575c22e6301ea6244/charts/opensearch/values.yaml#L324-L329

this is not referenced anywhere in [`statefulset.yaml`](https://github.com/opensearch-project/helm-charts/blob/63847ee78834063f5f79d2f575c22e6301ea6244/charts/opensearch/templates/statefulset.yaml).

**To Reproduce**
see above

**Expected behavior**
* there are no unused definitions in `values.yaml`
* the `StatefulSet` has a `readinessProbe` (and, preferably, also a `livenessProbe`; this that could be split into another issue)

**Chart Name**
opensearch

**Screenshots**
n/a

**Host/Environment (please complete the following information):**
n/a

**Additional context**
n/a

**Comment by peterzhuamazon** [comment]
@DandyDeveloper @TheAlgo is this something we missed at any point?
Thanks.

**Comment by TheAlgo** [comment]
Yes `liveness` probe needs to be removed until we have something in place. I have a pending PR on me for `readinessProbe` #172 . Will try to complete it asap.
